### ISSUE TITLE
feat(a11y): Add keyboard accessibility to AT unit home

### ISSUE THREAD

**Issue by breity** [WISE-Community/WISE-Client#1942]
## Description
Some action elements in the Authoring Tool unit home lessons and steps are not keyboard accessible. All elements should be navigable and selectable using the keyboard.
